src/components/Event.js

```diff
@@ -0,0 +1,14 @@
+function subscribe(eventName, listener) {
```

Comment:
I dont think we would need to communicate with custom events. We can do this via Parent - Child Component interaction

Reply:
Using a custom event for setting the timer. 
Unable to reset timer properly via  Parent - Child.